src/modules/edc-demo/components/contract-definition-editor-dialog/contract-definition-editor-dialog.component.ts

```diff
@@ -79,8 +79,12 @@ export class ContractDefinitionEditorDialog implements OnInit, OnDestroy {
           this.close({refreshList: true});
         },
         error: (error) => {
-          console.error('Failed creating asset!', error);
-          this.notificationService.showError('Failed creating policy!');
+          if(error.status == 409){
```

Comment:
not formatted correctly. ensure your intellij has the prettier plugin and uses prettier format on save